blockly-core/build-output/blockly_uncompressed.js

```diff
@@ -23437,7 +23437,7 @@ Blockly.BlockSpaceEditor.prototype.createDom_ = function(container) {
   svg.appendChild(this.blockSpace.createDom());
```

Comment:
I did; I figure we usually want to check these in if we have an intentional change to blockly_core, so that not everybody has to rebuild blockly_core after a pull.